drivers/hmis/lib/form_data/static/client_alert.json

```diff
@@ -34,7 +34,7 @@
           "id": "min-expiration",
           "type": "MIN",
           "_comment": "entry date cannot be in the past",
-          "value_local_constant": "$today"
+          "value_local_constant": "$tomorrow"
```

Comment:
You can actually do this with  `"value_local_constant": "$today", "offset": 1` already. We needed this to specify some different bounds that are needed around 7-14-30 days in the future or after entry/exit, and so on.
https://github.com/greenriver/hmis-warehouse/blob/860b84166a18bbe6993d1052e386bdf929ff847f/drivers/hmis/lib/form_data/allegheny/records/referral_request.json#L27